docs/shared/nx-plugin.md

```diff
@@ -0,0 +1,248 @@
+# Creating a Nx Plugin
+
+## Generating a Plugin
+
+To get started with building a Nx Plugin, you can run the following command:
+
+```bash
+npx create-nx-plugin my-org --pluginName my-plugin
+```
+
+This command will create a brand new workspace, and set up a pre-configured plugin with the specified name.
```

Comment:
Use present tense instead of future tense.

```suggestion
This command creates a brand new workspace, and sets up a pre-configured plugin with the specified name.
```